ansible/roles/software/ckan/catalog/ckan-app/templates/catalog-next/etc_ckan_production.ini.j2

```diff
@@ -0,0 +1,243 @@
+#		same origin policy
+ckan.plugins = {{ (catalog_ckan_plugins_default + catalog_ckan_plugins_additional) | join(' ') }}
+
+# Define which views should be created by default
+# (plugins must be loaded in ckan.plugins)
+#ckan.views.default_views = image_view text_view recline_view
+
+
+## Front-End Settings
+ckan.site_title = CKAN
+ckan.site_logo = /images/logo-geodata-small.png
+ckan.site_description =
+ckan.favicon = /fanstatic/datagovtheme/images/favicon.ico
+ckan.gravatar_default = identicon
+ckan.preview.direct = png jpg gif
+ckan.preview.loadable = html htm rdf+xml owl+xml xml n3 n-triples turtle plain atom csv tsv rss txt json
+
+# package_hide_extras = for_search_index_only
+#package_edit_return_url = http://another.frontend/dataset/<NAME>
+#package_new_return_url = http://another.frontend/dataset/<NAME>
+#ckan.recaptcha.version = 1
+#ckan.recaptcha.publickey =
+#ckan.recaptcha.privatekey =
+#licenses_group_url = http://licenses.opendefinition.org/licenses/groups/ckan.json
+#ckan.template_footer_end =
+
+ckan.template_head_end = <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
+
+## Internationalisation Settings
+ckan.locale_default = en
+ckan.locale_order = en pt_BR ja it cs_CZ ca es fr el sv sr sr@latin no sk fi ru de pl nl bg ko_KR hu sa sl lv
+ckan.locales_offered =
+ckan.locales_filtered_out = en_GB
+
+## Feeds Settings
+
+ckan.feeds.authority_name =
+ckan.feeds.date =
+ckan.feeds.author_name =
+ckan.feeds.author_link =
+
+## Storage Settings
+
+ckan.storage_path = /var/lib/ckan
+#ckan.max_resource_size = 10
+#ckan.max_image_size = 2
+
+# Resource Proxy settings
+# Preview size limit, default: 1MB
+ckan.resource_proxy.max_file_size = 1024 * 1024 * 50
+
+## Activity Streams Settings
+
+#ckan.activity_streams_enabled = true
+#ckan.activity_list_limit = 31
+#ckan.activity_streams_email_notifications = true
+#ckan.email_notifications_since = 2 days
+ckan.hide_activity_from_users = %(ckan.site_id)s
+
+
+## Email settings
+
+#email_to = you@yourdomain.com
+#error_email_from = paste@localhost
+#smtp.server = localhost
+#smtp.starttls = False
+#smtp.user = your_username@gmail.com
+#smtp.password = your_password
+smtp.mail_from = no-reply@data.gov
+
+ckan.tracking_enabled = False
+set debug = False
+
+## SAML2 Settings
+saml2.site_url = {{ saml2_site_url }}
+saml2.config_path = %(here)s/saml2
+
+saml2.user_mapping =
+    email~maxemail
+    fullname~field_display_name
+    id~maxid
+    name~maxemail
+
+saml2.organization_mapping =
+    name~field_unique_id
+    title~field_organization
+    extras:organization_type~field_organization_type
+
+saml2.max_redirect_slo =
+saml2.max_security_level = https://max.gov/icam/2015/10/securityLevels/securePlus2
+
+# ckanext-geodatagov settings
+
+ckanext.geodatagov.bureau_csv.url=https://project-open-data.cio.gov/data/omb_bureau_codes.csv
+ckanext.geodatagov.bureau_csv.url_default=https://project-open-data.cio.gov/data/omb_bureau_codes.csv
```

Comment:
FYI, project-open-data is going away and should be replaced by resources.data.gov. I opened this to track https://github.com/GSA/datagov-deploy/issues/1693